### DISCUSSION TITLE
Is LOTR big or little endian?

### DISCUSSION THREAD

**Discussion by DL8** [amichai-bd/riscv-multi-core-lotr#45] [Q&A] [answered]
Is LOTR big or little endian? Both options are fine from software perspective, but it should be documented

**Comment by amichai-bd** [comment, marked as answer]
Little endian.
Example:
Data[100] = 32'hDEAD_BEEF

Data[100] = 8'hEF
Data[101] = 8'hBE
Data[102] = 8'hAD
Data[103] = 8'hDE
